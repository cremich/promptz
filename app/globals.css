pre,
code {
  word-wrap: break-word !important;
  white-space: pre-wrap !important;
  white-space: -moz-pre-wrap !important;
  white-space: -pre-wrap !important;
  white-space: -o-pre-wrap !important;
  margin: 0;
  padding: 0;
}

code {
  background-color: #f3f3f7;
}

pre > code {
  display: inline-block;
  padding: 1em;
}
