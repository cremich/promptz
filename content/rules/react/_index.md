---
title: "React"
description: "Rules for React development, component patterns, and best practices."
---
