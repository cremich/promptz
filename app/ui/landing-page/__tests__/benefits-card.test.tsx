import { describe, expect, test } from "@jest/globals";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import BenefitCard from "@/app/ui/landing-page/benefits-card";
import { Code } from "lucide-react";

describe("Benefits Card", () => {
  test("renders benefits card unchanged", () => {
    const { container } = render(
      <BenefitCard
        icon={Code}
        title="Ready-to-Use Solutions"
        content=" Simplify software development with our curated collection of
        production-ready prompt templates"
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
