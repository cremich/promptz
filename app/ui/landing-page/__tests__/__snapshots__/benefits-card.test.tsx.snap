// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Benefits Card renders benefits card unchanged 1`] = `
<div>
  <div
    class="rounded-lg border bg-neutral-800 text-card-foreground shadow-sm text-center"
  >
    <div
      class="flex flex-col space-y-1.5 p-6"
    >
      <div
        class="text-2xl font-semibold leading-none tracking-tight items-center flex flex-col"
      >
        <svg
          class="lucide lucide-code w-12 h-12 text-violet-500 mb-4"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <polyline
            points="16 18 22 12 16 6"
          />
          <polyline
            points="8 6 2 12 8 18"
          />
        </svg>
        Ready-to-Use Solutions
      </div>
    </div>
    <div
      class="p-6 pt-0 text-zinc-400"
    >
       Simplify software development with our curated collection of production-ready prompt templates
    </div>
  </div>
</div>
`;
